src/containers/MapExplorer.jsx

```diff
@@ -57,7 +56,7 @@ class MapExplorer extends Component {
             {this.props.selectors.map((selector) => {
               selector.tmp = selector.id + selector.id;
               return (
-                <SelectorPanel ref={'selectorPanel_' + selector.id} key={selector.id} selectorData={selector} />
+                <MapControls ref={'mapControls_' + selector.id} key={selector.id} selectorData={selector} />
```

Comment:
string interpolation!

```
<MapControls ref={`mapControls_${selector.id}`} key={selector.id} selectorData={selector} />
```

not a huge deal in this case, but gets a lot easier to read with longer statements